pimegaApp/src/pimegaDetector.cpp

```diff
@@ -391,16 +403,7 @@ void pimegaDetector::captureTask() {
         backendStatus != 0 permits that the thread executes this snippet the
        last time when the NDFileCapture is set to 0 */
 
-    received_acq = 0;
-    for (int module = 1; module <= pimega->max_num_modules; module++) {
-      if (received_acq == 0 ||
-          (int)pimega->acq_status_return.STATUS_NOOFACQUISITIONS[module - 1] > received_acq) {
-        received_acq = (int)pimega->acq_status_return.STATUS_NOOFACQUISITIONS[module - 1];
-        if (received_acq < (int)pimega->acq_status_return.processedImageNum) {
-          received_acq = (int)pimega->acq_status_return.processedImageNum;
-        }
-      }
-    }
+    received_acq = receivedImagesAllModules(pimega);
```

Comment:
I'd add an explicit mention in the commit message of why this change also makes sense for this case. After all, it is not changing the `NumImagesCounter_RBV` PV, but rather comparing the `received_acq` value against `acquireParam.numCapture`.

(I'm just saying this because your commit message focuses only on the PV ;)